packages/react/src/select/root/useSelectRoot.ts

```diff
@@ -360,6 +363,11 @@ export namespace useSelectRoot {
      * The transition status of the Select.
      */
     transitionStatus?: TransitionStatus;
+    /**
+     * Whether the select is modal.
+     * @default true
+     */
```

Comment:
I've just added a new description for the modal prop across the board
https://github.com/mui/base-ui/blob/master/packages/react/src/dialog/root/useDialogRoot.ts#L167-L171

Reply:
:+1: Updated